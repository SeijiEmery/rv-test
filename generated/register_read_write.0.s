
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
