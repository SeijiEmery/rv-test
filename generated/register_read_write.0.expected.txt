tests/register_read_write.test.s:2
R0 = 0
R1 = 22
R3 = 44
R5 = 99
R31 = 22
R15 = 18446744073709534959
